File: Python/p11713.py
def isVowel (c):
    return c == 'a' or c == 'i' or c == 'u' or c == 'e' or c == 'o'

tc = int(input())
for i in range(0,tc):
    first = input()
    second = input()
    if len(first) != len(second):
        print("No")
    else:
        status = True
        pjg = len(first)
        for j in range(0,pjg):
            if not isVowel(first[j]) or not isVowel(second[j]):
                if first[j] != second[j]:
                    print("No")
                    status = False
                    break;
        
        if status:
            print("Yes")

File: Java/p11713.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class p11713 {
	public static void main (String [] args) throws IOException {
		BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
		int tc = Integer.parseInt(input.readLine());
		for(int i = 0; i < tc; i++) {
		    String first = input.readLine();
		    String second = input.readLine();
		    if (first.length() != second.length()) {
		        System.out.println("No");
		    } else {
				int length = first.length();
				boolean status = true;
				for (int j = 0; j < length; j++) {
				    if (!isVowel(first.charAt(j)) || !isVowel(second.charAt(j))) {
				        if (first.charAt(j) != second.charAt(j)) {
				            System.out.println("No");
				            status = false;
				            break;
				        }
				    }
				}
				
				if (status) {
				    System.out.println("Yes");
				}
			}
		}
	}
	
	static boolean isVowel(char c) {
	    return c == 'a' || c == 'i' || c == 'u' || c =='e' || c == 'o';
	}
}